docs/readthedocs/models/thermal_grid_model.md

```diff
@@ -3,6 +3,8 @@
 
 The Thermal Grid Model introduces a coupling point to thermal system, equivalent to an electrical node. It can be used to interconnect thermal units to a thermal heat network.
 
+A Thermal Grid Model consists of a {ref}`thermal_house_model` and / or a {ref}`cts_model` which are supplied by a thermal source like a [Heat pump].
```

Comment:
Link to heat pump does not work. 

Also, I think the thermal grid can really be configured to have no house, but in simulation this does not work, right? Isn't the house then (currently) mandatory?

Reply:
Simulation will start running but the hp is never used since there is no heat demand. So it's currently kind of mandatory since it's the only heat sink we have...